python-checks/src/main/java/org/sonar/python/checks/regex/UnquantifiedNonCapturingGroupCheck.java

```diff
@@ -19,16 +19,49 @@
  */
 package org.sonar.python.checks.regex;
 
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import javax.annotation.Nullable;
 import org.sonar.check.Rule;
+import org.sonar.plugins.python.api.quickfix.PythonQuickFix;
+import org.sonar.plugins.python.api.quickfix.PythonTextEdit;
 import org.sonar.plugins.python.api.tree.CallExpression;
+import org.sonar.python.regex.PythonRegexIssueLocation;
+import org.sonarsource.analyzer.commons.regex.RegexIssueLocation;
 import org.sonarsource.analyzer.commons.regex.RegexParseResult;
+import org.sonarsource.analyzer.commons.regex.ast.NonCapturingGroupTree;
+import org.sonarsource.analyzer.commons.regex.ast.RegexSyntaxElement;
 import org.sonarsource.analyzer.commons.regex.finders.UnquantifiedNonCapturingGroupFinder;
 
 @Rule(key = "S6395")
 public class UnquantifiedNonCapturingGroupCheck extends AbstractRegexCheck {
+  public static final String QUICK_FIX_MESSAGE = "Unwrap this unnecessarily grouped subpattern";
```

Comment:
I'd suggest `Unwrap subpattern` for the quick fix message.

Reply:
done